app/javascript/dashboard/routes/dashboard/settings/sla/components/Toggle.vue

```diff
@@ -0,0 +1,59 @@
+<script setup>
+import { ref } from 'vue';
+
+const props = defineProps({
+  value: {
+    type: Boolean,
+    default: false,
+  },
+});
+
+const emit = defineEmits(['input']);
+const isChecked = ref(props.value);
+
+const toggleCheckbox = () => {
+  isChecked.value = !isChecked.value;
+  emit('input', isChecked.value);
+};
+</script>
+
+<template>
+  <label class="relative inline-block h-4 w-7">
+    <input
+      class="hidden"
+      type="checkbox"
+      :checked="isChecked"
+      @click="toggleCheckbox"
+    />
+    <div
+      class="absolute inset-0 p-0.5 transition-all ease-in-out duration-200 rounded-full cursor-pointer slider bg-slate-100"
+    />
+  </label>
+</template>
+
+<style>
```

Comment:
I would suggest referring to https://github.com/shadcn-ui/ui/blob/main/apps/www/registry/default/ui/switch.tsx instead of writing the custom CSS here.

Reply:
I can move most of the styles to tailwind classes, I'll make the change!